### DISCUSSION TITLE
Recovery Key incorrect WHEN ITS LITERALLY COPY AND PASTE THE RECOVERY KEY

### DISCUSSION THREAD

**Discussion by bepikz** [ente-io/ente#2812] [Q&A]
I have zero idea why this is on git hub and for community help but im trying to sync my Arch Linux system with my phone for Ente Auth but apparently my computer is "not powerful enough to verify my password" and proceed me to the recovery key page in which it doesnt even work.

**Comment by nbari** [comment]
Same here, after installing with `paru -Sy ente-auth` and trying to use the Desktop version, I type the password but then I get this message:

```
Recreate Password

The current device is not powerful
enough to verify your password,
but we can regenerate in a way
that works with all devices.

Please login using your recovery 
key and regenerate your password
(you can use the same one again if 
you with).

Use recovery key
Cancel
```

I click in Use recovery key, type my recovery key, and get the error `Incorrect recovery key`

**Comment by nbrownuk** [comment]
The ente-auth package didn't work for me either, with the same issues you have suffered. However, the [ente-auth-bin](https://aur.archlinux.org/packages/ente-auth-bin) package seems to work just fine.
